src/fluxpy/utils/model.py

```diff
@@ -419,221 +421,307 @@ def _check_if_modelseed_model(model: cobra.Model):
 
 
 
-
-bigg_cofactors = ['atp[c]', 'atp_c', 'adp_c', 'adp[c]',
-                  'atp[c]', 'atp_c', 'adp_c', 'adp[c]',
-                  'udp[c]', 'udp_c', 'ump[c]', 'ump_c',
-                  'amp_c', 'amp[c]',
-                  'gdp[c]', 'gdp_c', 'gtp[c]', 'gtp_c',
-                  'accoa_c', 'accoa[c]', 'coa_c', 'coa[c]',
-                  'q8[c]', 'q8_c', 'q8h2_c', 'q8h2[c]', 'mqn8_c', 'mqn8[c]', 'mql8_c', 'mql8[c]', 'q8h2_c', 'q8h2[c]',
-                  'actp[c]', 'actp_c',
-                  'h2o_c', 'h2o[c]', 'h2o_e', 'h2o[e]',
-                  'pi_e', 'pi[e]', 'pi_c', 'pi[c]', 'ppi[c]', 'ppi_c',
-                  'pep_c', 'pep[c]',
-                  'h_c', 'h[c]', 'h_e', 'h[e]',
-                  'o2_c', 'o2[c]', 'o2_e', 'o2[e]',
-                  'co2_c', 'co2[c]', 'co2_e', 'co2[e]',
-                  'nadp_c', 'nadp[c]', 'nadph_c', 'nadph[c]', 'nad_c', 'nad[c]', 'nadh_c', 'nadh[c]',
-                  'nadp_e', 'nadp[e]', 'nadph_e', 'nadph[c]', 'nad_e', 'nad[e]', 'nadh_e', 'nadh[e]',
-                  'fadh2_c', 'fadh2[c]', 'fad_c', 'fad[c]',
-                  'nh4_c', 'nh4[c]', 'nh4_e', 'nh4[e]',
-                  'pyr[c]', 'pyr_c'
-                ]
-
-bigg_building_blocks = ['ala_L[c]', 'asp_L[c]', ' gln_L[c]', 'glu_L[c]', 'glu_L[c]', 
-                        'ser_L[c]', 'trp_L[c]', 'met_L[c]', 'lys_L[c]', 'cyst_L[c]'
-                       ]
-
-
-def find_arrow_index():
-    # load cobra model
-    cobra_model = load_json_model("../data/e_coli_core.json")
-    # list with reactions names
-    reactions_ids =  [ reaction.id for reaction in cobra_model.reactions ]
-    
-    # define arrow types
-    arrow_types = ["-->", "<=>", "<--"]
-    # define list to store indices
-    arrow_indices = []
+class Cofactor_Specificity:
```

Comment:
Since you re using the camel case for the class name and you also have a list called `cofactor_specificity` it is better to rename your class as `CofactorSpecificity`. 

Also, please add a description on its aim and maybe how to initiate an instance of it.